vault/resource_kv_secret_v2.go

```diff
@@ -284,22 +309,10 @@ func kvSecretV2Read(_ context.Context, d *schema.ResourceData, meta interface{})
 			return nil
 		}
 
-		log.Printf("[DEBUG] secret: %#v", secret)
-
-		data := secret.Data["data"]
-		jsonData, err := json.Marshal(data)
-		if err != nil {
-			return diag.Errorf("error marshaling JSON for %q: %s", path, err)
-		}
-
-		if err := d.Set(consts.FieldDataJSON, string(jsonData)); err != nil {
```

Comment:
TFVP will no longer track and set the sensitive data from Vault, maintaining the ephemerality of the new write-only attributes

Reply:
If I am understanding correctly, we will have to do this type of behavior change for all resources in which we implement as ephemeral types. If yes, that is a big behavior change for existing users.